### ISSUE TITLE
Some game icons are broken on the website

### ISSUE THREAD

**Issue by JayFoxRox** [citra-emu/citra-games-wiki#113]
This probably has to do with the icon.png being "16-bit sRGB" instead of "8-bit sRGB" as with most other files.

---

**See (Left = As seen on Website / Right = As stored in this repo):**

![Tomodachi Life Website](https://citra-emu.org/images/game/icons/tomodachi-life.png) ![Tomodachi Life GitHub](https://github.com/citra-emu/citra-games-wiki/blob/master/games/tomodachi-life/icon.png?raw=true)
![Spirit of Justice Website](https://citra-emu.org/images/game/icons/phoenix-wright-ace-attorney-spirit-of-justice.png) ![Spirit of Justice GitHub](https://github.com/citra-emu/citra-games-wiki/blob/master/games/phoenix-wright-ace-attorney-spirit-of-justice/icon.png?raw=true)

---

I'm not sure if we should (A) fix the validation script, (B) change the contribution guidelines or (C) fix the website (which probably converts these images poorly).
Maybe we should even do all of these.

**Comment by chris062689** [comment]
Good catch. We should fix the validation script to detect these, and fix the existing errors at the repo level. 

I feel the fix shouldn't be in the website, it should be self-contained in the repository since we may have other sources look at the repo. 

I think adding validation to the script is the right way to go about it.
